pkg/csi_driver/node.go

```diff
@@ -114,6 +127,7 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 			return nil, status.Errorf(storage.ParseErrCode(err), "failed to get GCS bucket %q: %v", bucketName, err)
 		}
 	}
+	vs.bucketAccessCheckPassed = true
```

Comment:
based on the comment https://github.com/GoogleCloudPlatform/gcs-fuse-csi-driver/pull/306/files#r1688774935, this can move after L128

Reply:
Done.